packages/nuxt/src/index.ts

```diff
@@ -73,30 +74,58 @@ export default defineNuxtModule<UnocssNuxtOptions>({
       })
     }
 
-    const { config: unoConfig } = await loadConfig<UserConfig>(process.cwd(), {
-      configFile: options.configFile,
-    }, [], options)
-
-    await nuxt.callHook('unocss:config', unoConfig)
-
-    if (
-      isNuxt3()
-      && nuxt.options.builder === '@nuxt/vite-builder'
-      && nuxt.options.postcss.plugins.cssnano
-      && unoConfig.transformers?.some(t => t.name === '@unocss/transformer-directives' && t.enforce !== 'pre')
-    ) {
-      const preset = nuxt.options.postcss.plugins.cssnano.preset
-      nuxt.options.postcss.plugins.cssnano = {
-        preset: [preset?.[0] || 'default', Object.assign(
-          { mergeRules: false, normalizeWhitespace: false, discardComments: false },
-          preset?.[1],
-        )],
+    if (options.nuxtLayers) {
+      addTemplate({
```

Comment:
Do we need a template for that? We could probably just read configs from all layers and merge them directly in the module and passed the merged object to Unocss?

Reply:
But in this case we will not be able tmake it work with Eslint. As I know there is no other way for eslint plugin to read Uno config

Reply:
@antfu Do you see any downsides to generating uno.config file? I think we should not leave out the support of Eslint.